packages/blockchain-wallet-v4-frontend/src/scenes/Transactions/Content/Empty/index.js

```diff
@@ -2,13 +2,39 @@ import React from 'react'
 import { connect } from 'react-redux'
 import { bindActionCreators } from 'redux'
 import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { FormattedMessage } from 'react-intl'
 
+import { Text } from 'blockchain-info-components'
 import { actions } from 'data'
-import Empty from './template.js'
+import CoinWelcome from './CoinWelcome'
 
+const Wrapper = styled.div`
+  width: 100%;
+  height: 90%;
+  padding: 30px;
+  box-sizing: border-box;
+`
+const NoBsv = styled.div`
+  margin: 20px;
+`
 class EmptyContainer extends React.PureComponent {
   render () {
-    return <Empty coin={this.props.coin} />
+    const { coin, handleRequest } = this.props
+    return coin === 'BSV' ? (
```

Comment:
Dont really want to do this

Reply:
Another way is to hide the whole thing based on the number of txs right, from the bsv setting page selector